### ISSUE TITLE
Update to new gov-uk colours

### ISSUE THREAD

**Issue by DaveHudson** [govuk-react/govuk-react#705]
The gov-uk colours have recently been updated https://designnotes.blog.gov.uk/2019/07/29/weve-updated-the-gov-uk-colours-and-font/ to meet WCAG 2.1 level AA.

Wondering if there are plans to update the colours in govuk-react to match? Happy to have a crack at creating an PR to update the colours if that helps. From what I can see it would need an PR with the new colours in https://github.com/penx/govuk-colours and then an PR in this repo to update the version number of govuk-colours in package.json.

**Comment by jsrobertson** [comment]
Hi @DaveHudson - Please feel free to raise a PR in both repos.  @penx May need to approve in the govuk-colours repo but I can approve the package.json update on here when required.

It would be a big help if you could take a look, as we're aware that some updates need to be made to re-align with the recent govuk-frontend changes.

**Comment by DaveHudson** [comment]
@jsrobertson sure no problem. I'll have a look at creating PRs asap for both repos 👍

**Comment by penx** [comment]
@jsrobertson I've given you collaborator access on govuk-colours if you need it

**Comment by DaveHudson** [comment]
I've made the changes to `govuk-colours` locally and run against `govuk-react` so I can see the impact of the changes in this repo / test it out before making the PR in `govuk-colours`. 

58 snapshot tests will have to be updated due to these colour changes, which is fine. Reading the `CONTRIBUTING.md` file though it says:

`when fixing a visual issue, include a unit test or snapshot test that proves the issue, and include a screenshot of the component before and after in the PR`

That's a lot of screenshots to capture given it's 58 snapshot changes! Will the `chromatic-test` command do this for us automatically? 

Tried running locally but it doesn't work, assume Chromatic needs to run as part of CI or something...

**Comment by penx** [comment]
Don’t worry about attaching images to the pr for this. That note is more about where we’re manually changing something that would previously be considered incorrect or an intentional change to layout behaviour for a small number of components

**Comment by penx** [comment]
Also thanks for reading contributing.md to that detail 😄👌

**Comment by DaveHudson** [comment]
FYI: Created the PR in [GovUK-colours](https://github.com/penx/govuk-colours/pull/5)

Once that is in master will update package.json and snapshots for this PR

**Comment by penx** [comment]
v1.1.0 has been released to npm https://www.npmjs.com/package/govuk-colours

**Comment by penx** [comment]
Done in #768
